File: core/src/main/java/com/mobilesolutionworks/android/app/WorkControllerHost.java
package com.mobilesolutionworks.android.app;

import com.mobilesolutionworks.android.app.controller.WorksControllerManager;

/**
 * Created by yunarta on 15/3/17.
 */
@FunctionalInterface
public interface WorkControllerHost {

    WorksControllerManager getControllerManager();
}

File: library/src/main/kotlin/com/mobilesolutionworks/android/app/WorkControllerHost.kt
package com.mobilesolutionworks.android.app

import com.mobilesolutionworks.android.app.controller.WorksControllerManager

/**
 * Created by yunarta on 15/3/17.
 */
@FunctionalInterface
interface WorkControllerHost {

    val controllerManager: WorksControllerManager
}
